### ISSUE TITLE
Would you make it possible to develop a trump2cash for FX trade as well?

### ISSUE THREAD

**Issue by cyouraku** [maxbbraun/trump2cash#45]
Trump says so much about FX, hope it works on FX trading as well.

**Comment by moe-khan123** [comment]
Good idea. I was looking through Trump's twitter feed when he was planning to place tariffs on Canada's lumber imports. When that announcement was made it really plummeted Canada's Dollar which would have made a nice short selling opportunity. I think every time he mentions imposing a tax on a country it would be a nice trading opportunity.

**Comment by WilliamJamesDalrympleWest** [comment]
Did anyone ever do this?
